libcontainer/nsenter/README.md

```diff
@@ -13,13 +13,7 @@ So every time we  import package `nsenter`, the C code function `nsexec()` would
 called. And package `nsenter` is now only imported in `main_unix.go`, so every time
 before we call `cmd.Start` on linux, that C code would run.
 
-`nsexec()` will first check the environment variable `_LIBCONTAINER_INITPID` 
-which will give the process of the container that should be joined. Namespaces fd will 
-be found from `/proc/[pid]/ns` and set by `setns` syscall.
-
-And then get the pipe number from `_LIBCONTAINER_INITPIPE`, error message could
-be transferred through it. If tty is added, `_LIBCONTAINER_CONSOLE_PATH` will 
-have value and start a console for output.
-
-Finally, `nsexec()` will clone a child process , exit the parent process and let 
-the Go runtime take over.
+`nsexec()` will first get the init pipe number from the  environment variable
+`_LIBCONTAINER_INITPIPE`. The init pipe is then used to read bootstrap data
+(e.g., namespaces, cloneflags, uidmap, gidmap, and consolefd), and to write pid
+to after nsexec finishes setting up the environment.
```

Comment:
I should be more specific:

    `nsexec()` will first get the init pipe number from the  environment variable
    `_LIBCONTAINER_INITPIPE`. The init pipe is then used to read bootstrap data
    (e.g., namespaces, cloneflags, uidmap, gidmap, and consolefd) from the runc
    process, and to send the pid of the child process back to the runc process
    after nsexec clones a child process.